tests/dbgp/dbgpclient.php

```diff
@@ -1,16 +1,53 @@
 <?php
 class DebugClient
 {
+	protected $triedPorts = 0;
+
+	protected $startPort = 9991;
```

Comment:
If the port wasn't released, that means there is another bug really. I don't particularly *mind* this change, but it should be in a separate commit to keep things separated and clear. For now, I'm inclined to not want to have that in this already slightly complicated patch.